megamek/src/megamek/common/BombType.java

```diff
@@ -458,7 +458,7 @@ private static BombType createLargeFuelAirBomb() {
         BombType bomb = new BombType();
 
         bomb.name = "Fuel-Air Bomb (Large)";
-        bomb.shortName = "FAEBomb_l";
+        bomb.shortName = "FAE Bomb (l)";
```

Comment:
Maybe use capital letter instead of lowercase here? l, 1, I, and | can be annoying to tell the difference between them, but L is easy to tell.